sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -874,6 +874,7 @@ Mono<Response<ConfigurationSetting>> setReadOnly(ConfigurationSetting setting, b
      * @param selector Optional. Selector to filter configuration setting results from the service.
      * @return A Flux of ConfigurationSettings that matches the {@code selector}. If no options were provided, the Flux
      * contains all of the current settings in the service.
+     * @throws HttpResponseException if there is a client or service error occurs, as in 400 or 500 error types.
```

Comment:
Minor rewording as 4xx and 5xx are service errors and not client errors.

```suggestion
     * @throws HttpResponseException If a non-recoverable service error occurs, such as a 404 or 409.
```